docs/js/modules/badges.js

```diff
@@ -0,0 +1,35 @@
+
+export const badgesList = {
+   'top-beta': [
+      'Billionaire during the first beta',
+      'top-beta.png'
+   ],
+   'top-s1': [
+      'the richest investor on memeEconomy',
+      'top-investor.png'
+   ],
+   'contributor': [
```

Comment:
I have suggestions for the badge descriptions:

- top-beta: "Beta Billionaire: Earned a billion MemeCoins or more during the beta"

- top-s1: "S1 Top Investor: Reached the height of riches in Season 1"

- contributor: "Contributor: Made at least one commit to the MemeInvestor_bot project on GitHub"

- overflow: "Overflow: 'The value of his bank account is too damn high!' "